File: Linguagens_Programação/C/2.c
//2. Dado um inteiro positivo n, verificar se n é primo.
#include <stdio.h>
int main()
{
    int N;
    int cont = 0;
    printf("Digite um inteiro N: ");
    scanf("%i",&N);
   
    for (int i=1; i<=N; i++){
        if (N % i == 0){
            cont++;
        }
    }

    if (cont == 2){
        printf("O número %i é primo!",N);
    }
    else{
        printf("O número %i não é primo!",N);
    }

    return 0;
}

File: Linguagens_Programação/Go/2.go
/*2. Dado um inteiro positivo n, verificar se n é primo. */
package main
import "fmt"

func main() {
    var n,totalMult int
    
    fmt.Println("Digite um inteiro N: ")
    fmt.Scanln(&n)
    totalMult = 0
    
    for i := 1; i <= n; i++ {
        if n%i==0{
            totalMult++
        }
        
    }
    if totalMult == 2{
        fmt.Printf("Eh primo")
        
    } else {
        fmt.Printf("Não eh primo")
    }

}